### PR TITLE
What's New for engine 2.5

### PR DESCRIPTION
**Pull request by micheles**
See https://github.com/gem/oq-engine/blob/whatsnew/doc/whats-new.md

### CODE REVIEW

File: doc/whats-new.md [hunk 1]
```diff
@@ -1,497 +1,144 @@
...
+transfer. The computational performance is more or less the same as
+before, though.
+
+This is the first release integrating hazardlib into the engine: there
+are no more hazardlib packages. The change is transparent to the user
+and the upgrade procedure will automatically uninstall the old
+hazardlib. However, scientists and developers using the oq-hazardlib
+repository from GitHub should know that it has been deprecated.  In
+order to avoid confusion, we suggest to remove the oq-hazardlib
+repository if you have one.
```
Comment (daniviga) [inline doc/whats-new.md]: I would add link to https://github.com/gem/oq-engine/blob/master/doc/faq.md#openquke-hazardlib-errors `faq.md`

File: doc/whats-new.md [hunk 2]
```diff
@@ -1,497 +1,144 @@
...
+
+There were several improvements to the Web UI:
+
+- the command `oq webui start` now open a browser window, if possible;
+- we added the commands `oq webui createsuperuser` and `oq webui collectstatic`
+  which are useful for system administrators setting a multiuser instance of
+  the WebUI (see https://github.com/gem/oq-engine/blob/engine-2.5/doc/installing/server.md);
+- there is a confirmation dialog when removing a calculation;
+- we added an end point `v1/calc/XXX/oqparam` to extract the calculation
+  parameters as a JSON dictionary, which is used by the QGIS plugin.
```
Comment (daniviga) [inline doc/whats-new.md]: We may link 'QGIS Plugin' to https://plugins.qgis.org/plugins/svir/

**Comment by klunk386** [comment]
I don't understand the sentence "the upgrade procedure will automatically uninstall the old hazardlib". Is this referred just to Linux packages or there is some sort of universal uninstaller?

**Comment by daniviga** [comment]
I agree with @klunk386, that statement is misleading. I would specify if standalone installers or packages are used, otherwise I would point users to the FAQ (see my previous comment on that).

**Comment by klunk386** [comment]
LGTM

File: doc/whats-new.md [hunk 3]
```diff
@@ -15,12 +15,10 @@ transfer. The computational performance is more or less the same as
...
-are no more hazardlib packages. The change is transparent to the user
-and the upgrade procedure will automatically uninstall the old
-hazardlib. However, scientists and developers using the oq-hazardlib
-repository from GitHub should know that it has been deprecated.  In
-order to avoid confusion, we suggest to remove the oq-hazardlib
-repository if you have one.
+are no more hazardlib packages. The change is transparent to the users
+of the official packages/installers, i.e. there is nothing to do for them.
+However, scientists and developers using the oq-hazardlib repository
+from GitHub should [remove it manually](https://github.com/gem/oq-engine/blob/engine-2.5/doc/faq.md#openquake-hazardlib-errors) to avoid possible confusions.
```
Comment (daniviga) [inline doc/whats-new.md]: `from GitHub` -> `from GitHub or via pypi`

**Pull request merged**